Core/UIViewControllerExtension.swift

```diff
@@ -25,15 +25,23 @@ extension UIViewController {
     var isSmall: Bool {
         return view.frame.height <= 568
     }
+    
+    var isPad: Bool {
```

Comment:
previous global variable was not flexible enough when dealing with different size classes on iPad, so that has been removed and local isPad that meets the needs of the usage more appropriately have been added.

Reply:
Just realized it while I'm testing. Thanks for the clarification.